### ISSUE TITLE
Request for jax.scipy.special.airy to be implemented

### ISSUE THREAD

**Issue by elliottperryman** [jax-ml/jax#25244]
I would like to have a jax implementation of scipy.special.airy. I work with physics simulations, and this is a very common function to see. Right now, I want to differentiate a quantum mechanical simulation so that I can see how sensitive the simulation is and use gradient descent methods (and the time for a single simulation is large so that finite differences are too slow.

I checked for duplicate requests and don't see them.
